### PR TITLE
2.1 to master

### PR DESCRIPTION
**Pull request by joonsp**

### CODE REVIEW

File: Tests/bootstrap.php [hunk 1]
```diff
@@ -1,16 +0,0 @@
-<?php
```
Comment (janimatti) [inline Tests/bootstrap.php:1]: why deleted?

File: Tests/bootstrap.php [hunk 2]
```diff
@@ -1,16 +0,0 @@
...
-
-if (!@include __DIR__ . './../vendor/autoload.php') {
```
Comment (janimatti) [inline Tests/bootstrap.php:3]: why deleted?

File: Twig/Extensions/ConfirmDialog.php [hunk 3]
```diff
@@ -45,7 +45,7 @@ class ConfirmDialog extends \Twig_Extension
     /**
      * @param Twig_Environment $twig 
      */
-    public function __construct(Twig_Environment $twig, Router $router, $route, $template, $defaultArguments = array())
+    public function __construct(Twig_Environment $twig, RouterInterface $router, $route, $template, $defaultArguments = array())
```
Comment (janimatti) [inline Twig/Extensions/ConfirmDialog.php:52]: Check docblock

File: Tests/bootstrap.php (hunk 2 above)
Comment (joonsp) [inline Tests/bootstrap.php:3]: no phpunit tests

**Pull request merged**